tests/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriberTest.php

```diff
@@ -301,7 +305,7 @@ public function testResponseHasPromotion(bool $isSearch, string $endpoint): void
 
     public function testResponseHasNoPromotion(): void
     {
-        $this->configMock->expects($this->once())->method('isActive')->willReturn(true);
+        $this->configMock->expects($this->any())->method('isActive')->willReturn(true);
```

Comment:
Possible to use `->exactly()` here? Or is there a reason not to use it?

Reply:
I only want to tell the config once that it is active, if it is active, why should it not be active in the same test? Doesn't make sense, so I explicitly wrote `any`.

Reply:
Alright